pkg/arvo/app/hark-graph-hook.hoon

```diff
@@ -59,9 +60,21 @@
   |=  old=vase
   ^-  (quip card _this)
   :_  this(state !<(state-0 old))
-  ?:  (~(has by wex.bowl) [/graph our.bowl %graph-store])
-    ~
-  ~[watch-graph:ha]
+  %+  welp
+    ?:  (~(has by wex.bowl) [/graph our.bowl %graph-store])
+      ~
+    ~[watch-graph:ha]
+  %+  turn
```

Comment:
Perhaps graph-store should keep track of a list of validator marks currently in use, and provide subscription updates when that list changes? Would remove the need to hard code this sort of list. (Your use case of knowing when the marks change is righteous, though, certainly, and this is not a blocker, just a suggestion for the future)